[2022-08-16 16:43:44,643] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T16:43:09.838712+00:00 [queued]>
[2022-08-16 16:43:44,730] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T16:43:09.838712+00:00 [queued]>
[2022-08-16 16:43:44,733] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 16:43:44,734] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-16 16:43:44,735] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 16:43:44,865] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2022-08-16 16:43:09.838712+00:00
[2022-08-16 16:43:44,897] {standard_task_runner.py:52} INFO - Started process 1067 to run task
[2022-08-16 16:43:44,961] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2022-08-16T16:43:09.838712+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag.py', '--cfg-path', '/tmp/tmpv8rcq7b8', '--error-file', '/tmp/tmposrgy9do']
[2022-08-16 16:43:44,966] {standard_task_runner.py:80} INFO - Job 67: Subtask insert_data
[2022-08-16 16:43:46,015] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T16:43:09.838712+00:00 [running]> on host c6d8e67217b1
[2022-08-16 16:43:46,618] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-16T16:43:09.838712+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-16T16:43:09.838712+00:00
[2022-08-16 16:43:46,722] {logging_mixin.py:115} INFO - rate_date:  2022-08-16
[2022-08-16 16:43:46,815] {logging_mixin.py:115} INFO - value_:  24207.92857100000037
[2022-08-16 16:43:47,218] {base.py:68} INFO - Using connection ID 'postgre_test_db' for task execution.
[2022-08-16 16:43:47,360] {python.py:173} INFO - Done. Returned value was: None
[2022-08-16 16:43:47,468] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20220816T164309, start_date=20220816T164344, end_date=20220816T164347
[2022-08-16 16:43:47,646] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-16 16:43:48,720] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
